app/single_resource/views.py

```diff
@@ -14,8 +14,13 @@
 def index():
     """View resources in a list."""
     resources = Resource.query.all()
-    return render_template('single_resource/index.html', resources=resources)
+    return render_template('single_resource/index.html', resources=resources, query="")
```

Comment:
instead of passing an empty query parameter I think it would be better to check if query is defined in the template itself?